src/style.js

```diff
@@ -13,7 +13,7 @@ const style = css`
     --mmp-media-cover-info-color: var(--mini-media-player-media-cover-info-color, --mmp-text-color);
     --mmp-text-color-inverted: var(--disabled-text-color);
     --mmp-active-color: var(--mmp-accent-color);
-    --mmp-icon-color: var(--mini-media-player-icon-color, var(--paper-item-icon-color, #44739e));
+    --mmp-icon-color: var(--mini-media-player-icon-color, --mini-media-player-base-color, var(--paper-item-icon-color, #44739e));
```

Comment:
I think we have to modify this line a bit for it to work properly, we should probably nest the fallbacks (if more than one), see [this](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#Custom_property_fallback_values).

```js
--mmp-icon-color: var(--mini-media-player-icon-color, var(--mini-media-player-base-color, var(--paper-item-icon-color, #44739e)));
```

Reply:
my bad, sorry!
fixed in a new commit

Reply:
The nested syntax is pretty ugly and messy imo, the way you wrote it first does actually make more sense imo 🙂